pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -33,6 +34,9 @@ const (
 	SQSTerminateKind = "SQS_TERMINATE"
 )
 
+// ErrNodeTerminated forwards condition that the instance is terminated thus metadata missing
```

Comment:
should `ErrNodeTerminated` be `ErrNodeMetadataUnavailable` ? 

What do you think about renaming this something more specific to the node state not being "running" since that's what we're explicitly checking on line 198? Maybe `ErrNodeStateNotRunning` or `ErrNodeNotRunning` ?

Reply:
good point  - adjusted!